src/lib.rs

```diff
@@ -92,6 +92,9 @@ use lazy_static::lazy_static;
 #[cfg(feature = "metadata")]
 pub use crate::properties::*;
 
+/// Re-export of jack_sys, to ensure compatibility
+pub use jack_sys;
```

Comment:
What about `pub use jack_sys as ffi;`

Reply:
Adding the extra warning makes a lot of sense.  I’d be wary about confusion between `jack::ffi` and `std::ffi`, but it’s not a string objection: I’ll happily defer to your judgement as the package maintainer.

Reply:
This seems ok.